llama_index/llama_dataset/download.py

```diff
@@ -39,10 +41,13 @@ def download_llama_dataset(
         library_path="llama_datasets/library.json",
         disable_library_cache=True,
         override_path=True,
+        show_progress=show_progress,
     )
     rag_dataset_filename, source_filenames = filenames
 
     return (
         LabelledRagDataset.from_json(rag_dataset_filename),
-        SimpleDirectoryReader(input_files=source_filenames).load_data(),
+        SimpleDirectoryReader(
+            input_dir=download_dir, exclude=[LLAMA_RAG_DATASET_FILENAME, "__init__.py"]
```

Comment:
can add a list for such filenames, which include `__init__.py`

Reply:
yeah, do you mean add a constant list like EXCLUDE_FILES = [..., "__init__.py"]?

Reply:
yes a list for only exclude files so that we can maintain it easily